docs/logagent/output-plugin-http.md

```diff
@@ -0,0 +1,38 @@
+title: Ship logs via http or https
+description: Sends log messages as http or https post request
+
+## Output Plugin: HTTP
+
+Post logs to web services via `http` or `https`. 
+
+Supported formats: 
+- ldjson - line delimited json 
+
+### Configuration
+
+```
+ output:
+    module: output-http
+    url: http://localhost:8080/events
+    format: ldjson
+    # maximum number of events to buffer
+    # 1 - each event creates a separate http request
+    # >1 - multiple events in each http request
+    maxBufferSize: 1
+    # flush interval in seconds
+    flushInterval: 1
+    # add tags before logs are shipped
+    tags:
+      token: SPM_TOKEN
```

Comment:
Does this work with MONITORING_TOKEN as well?

Reply:
Tags are just text fields, we can remove `toke`n from the example. So people are not confused with any of Sematext App tokens.